packages/@aws-cdk/aws-codestarnotifications-targets/lib/slack-channel-configuration.ts

```diff
@@ -0,0 +1,22 @@
+import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as notifications from '@aws-cdk/aws-codestarnotifications';
+
+/**
+ * A Slack notification target
+ */
+export class SlackChannelConfiguration implements notifications.IRuleTarget {
+
+  /**
+   * @param slackChannel The Slack channel configuration
+   */
+  constructor(readonly slackChannel: chatbot.ISlackChannelConfiguration) {}
+
+  public bind(
+    _rule: notifications.IRule,
+  ): notifications.RuleTargetConfig {
```

Comment:
What's with this formatting...?

```suggestion
  public bind(_rule: notifications.IRule): notifications.RuleTargetConfig {
```

Reply:
I will fix this formatting.